### DISCUSSION TITLE
pycharm copilot can not login(tried proxy also)

### DISCUSSION THREAD

**Discussion by zArkyd** [community/community#14761] [Copilot Conversations] [answered]
by click login github under tool in pycharm shows nothing after "Retrieving Github Device Code", no pop-up sad````

**Comment by jansorg** [comment, marked as answer]
This could be an issue with the browser setup in your IDE or system.
You can always 
1. Copy the code from the dialog
2. open the URL manually, https://github.com/login/device
3. Click "Ok" in the dialog to make it wait for device authentication
4. Paste and confirm the code on the GitHub URL above

**Reply by zArkyd** [reply]
solved, thanks alot!

**Reply by tohka0x01** [reply]
> solved, thanks alot!

您好，跟您的情况是一样的，您是如何拿到code的呢？

**Reply by zArkyd** [reply]
> > solved, thanks alot!
> 
> 您好，跟您的情况是一样的，您是如何拿到code的呢？

VPN打开然后pycharm里的代理选第二个，自动识别代理，然后开起pycharm之前VPN先开全局
